jetstream/probe_sets.py

```diff
@@ -139,55 +141,38 @@ def discriminate_telemetry(d, type):
             for klass in type.__args__:
                 if kind == klass.kind:
                     return klass(**d)
-            raise ValueError(f"Could not discriminate telemetry kind {kind}")
+            raise ValueError(f"Could not discriminate probe kind {kind}")
 
-        converter.register_structure_hook(FeatureTelemetryType, discriminate_telemetry)
+        converter.register_structure_hook(ProbeType, discriminate_telemetry)
         return converter.structure(d, cls)
 
 
-class ResolvesFeatures(Protocol):
-    def resolve(self, slug: str) -> Feature:
+class ResolvesProbeSets(Protocol):
+    def resolve(self, slug: str) -> ProbeSet:
```

Comment:
Having this protocol doesn't seem strictly necessary since we only have a single class that implements it. So we could also remove it?

Reply:
It's maybe useful if we want to type-check our tests: eh

Reply:
Yeah, possibly. We can leave it for now.